Makefile

```diff
@@ -226,10 +226,10 @@ clean:
 
 ci: ## Run unit tests and benchmark with different configurations
 ci:
-	@ocaml ./configure.ml && BENCH_NAME=vanilla $(MAKE) -s clean test bench-marshal clean
-	@ocaml ./configure.ml --sosa-num && BENCH_NAME=num $(MAKE) -s clean test bench-marshal clean
-	@ocaml ./configure.ml --sosa-zarith && BENCH_NAME=zarith $(MAKE) -s clean test bench-marshal clean
-	@$(MAKE) -s bench-tabulate
+	@ocaml ./configure.ml && BENCH_NAME=vanilla $(MAKE) -s clean test bench-marshal
+	@ocaml ./configure.ml --sosa-num && BENCH_NAME=num $(MAKE) -s clean test bench-marshal
+	@ocaml ./configure.ml --sosa-zarith && BENCH_NAME=zarith $(MAKE) -s clean test bench-marshal
+	@$(MAKE) -s clean bench-tabulate
```

Comment:
Here it's the same '''clean''' before bench-tabulate is not needed because bench-tabulate only format the stats contained in .bin (exe+bin do the job whatever the state of the rest of the repository).